src/main/java/com/google/devtools/build/lib/skylarkbuildapi/SkylarkActionFactoryApi.java

```diff
@@ -159,17 +159,29 @@
   @SkylarkCallable(
       name = "symlink",
       doc =
-          "<p><b>Experimental</b>. This parameter is experimental and may change at any "
```

Comment:
Why did you remove the "Experimental" annotation?

Reply:
I intend to only mark the "dangling symlink" behavior as experimental, the artifact -> artifact symlink introduced in this PR has been used by native rules for a long time and (IMO) shouldn't need to be behind a flag.

I agree that it would be nice to have an "Experimental" warning for the dangling symlink part somewhere, I just wasn't sure where to put that warning. OTOH, the only way to get a dangling symlink is `ctx.actions.declare_symlink()`, which already is marked as experimental, so 🤷‍♀️.